### PR TITLE
Separate log file for watcher thread

### PR DESCRIPTION
**Pull request by simathih**
- log watcher process logs into watcher.log instead of extension.log

### CODE REVIEW

**Comment by simathih** [comment]
I will remove python 3 usage in the next iteration

File: OmsAgent/Fairfax/omsagent_ff.py [hunk 1]
```diff
@@ -513,7 +513,11 @@ def parse_context(operation):
     if ('Utils.WAAgentUtil' in sys.modules
             and 'Utils.HandlerUtil' in sys.modules):
```
Comment (simathih) [inline OmsAgent/Fairfax/omsagent_ff.py]: I will remove python 3 usage if any in the next iteration

File: Utils/HandlerUtil.py [hunk 2]
```diff
@@ -81,14 +81,14 @@ def __init__(self,name):
         self._seq_no = -1
         self._status_file = None
         self._settings_file = None
-        self._config = None
+        self._config = None
```
Comment (boumenot) [inline Utils/HandlerUtil.py]: NIT: whitespace.

File: Utils/HandlerUtil.py [hunk 3]
```diff
@@ -81,14 +81,14 @@ def __init__(self,name):
         self._seq_no = -1
         self._status_file = None
         self._settings_file = None
-        self._config = None
+        self._config = None        
         return
 
 class HandlerUtility:
-    def __init__(self, log, error, s_name=None, l_name=None, extension_version=None):
+    def __init__(self, log, error, s_name=None, l_name=None, extension_version=None, fromTelemetry = False):
```
Comment (boumenot) [inline Utils/HandlerUtil.py]: Is this change really for all extensions, or just for OMS?  If it is just OMS, I wouldn't bother with anything else.

**Pull request merged**